x-pack/plugins/canvas/public/components/app/index.js

```diff
@@ -4,11 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import {
```

Comment:
@clintandrewhall could you check these changes, if they are not breaking any loading behavior in Canvas?